content/content.go

```diff
@@ -13,7 +13,7 @@ import (
 var (
 	errNotFound = errors.New("content: not found")
 
-	bufPool = sync.Pool{
+	BufPool = sync.Pool{
```

Comment:
@dmcgowan up to you and @stevvooe .  I just hacked it in because I needed to split out the service from the store.